### ISSUE TITLE
Instructions tell users to start rails with `rails s`

### ISSUE THREAD

**Issue by justin808** [shakacode/react_on_rails#1488]
### Discussed in https://github.com/shakacode/react_on_rails/discussions/1485

<div type='discussions-op-text'>

<sup>Originally posted by **jasonfb** September 23, 2022</sup>
Please note I am stuck getting the basic installation working so perhaps there is a gap in my understanding. please elucidate me if so.
see https://github.com/shakacode/react_on_rails/issues/1483 for my issue where I am stuck here.

however, I tried both `foreman start -f Procfile.dev-static` and also `rails s` and both produce the same results (see issue)

What I don't understand is if the webpack dev server is needed to be running in the background to make it work when compile is set to false (which the react_on_rails installer has just done in the prior step), why do these instructions tell the user to start the server with `rails s` and not `foreman start -f Procfile.dev-static` 

![Getting Started | Shakacode 2022-09-23 08-54-27](https://user-images.githubusercontent.com/59002/191965410-fbb07a61-330e-4cd6-bc5d-9b2ef80b3452.png)
</div>

**Comment by justin808** [comment]
https://github.com/shakacode/react_on_rails/issues/1488

FYI -- if webpacker.yml setting for `compile` is true, then the current instructions are correct. However, that's not the recommended path, which is to either the webpack-dev-server via `bin/webpacker-dev-server` or run `bin/webpacker --watch`. These are setup by the Procfile examples.

**Comment by mgmodell** [comment]
Without specifying any sort of starting point, these instructions do not work. I ran a `rails new` using the latest rails (7.0.4) and the defaults there do not include anything webpacker. Therefore, whether I launch with `rails s`, `foreman` or I specify that `compile` is true, it does not function.

I'll update the docs to specify the prerequisite of a rails app with webpacker enabled (i.e. `rails n _____ --javascript=webpack`) because [we've already got instructions](https://www.shakacode.com/react-on-rails/docs/guides/installation-into-an-existing-rails-app/) for adding it if it isn't already present.
